src/main/java/study/core/PlayerName.java

```diff
@@ -0,0 +1,22 @@
+package study.core;
+
+public class PlayerName {
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String name;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 공백 라인을 추가해보면 좋을 것 같습니다.
```suggestion
    private static final int MAX_NAME_LENGTH = 5;

    private final String name;
```